### PR TITLE
Create words learned filter function and user search function

### PR DESCRIPTION
**Pull request by hironaooo**

### CODE REVIEW

File: app/views/words/listEachClass.js.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+$("#classList").html("<%= escape_javascript(render 'classList_body', :majorID => @majorID, :universityID => @universityID) %>");
```
Comment (framgia1023) [inline app/views/words/listEachClass.js.erb]: Please remove this file.

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1,5 +1,4 @@
-<h1 class="text-center border-bottom p-4 mb-3">Users</h1>
-
+<h1 class="text-center border-bottom mb-3">Users</h1>
 <div class="row">
     <% @users.each do |user| %>
         <div class="col-lg-6">
```
Comment (framgia1023) [inline app/views/users/index.html.erb]: If user search result returns nothing, display "Nothing to display"

File: app/models/user.rb [hunk 3]
```diff
@@ -50,7 +50,11 @@ def following?(other_user)
 
   def self.search(search) #ここでのself.はUser.を意味する
     if search
-      where(['name LIKE ?', "%#{search}%"]) #検索とnameの部分一致を表示。User.は省略
+      if where(['name LIKE ?', "%#{search}%"]) == [] #検索とnameの部分一致を表示。User.は省略
+        
+      else 
+        where(['name LIKE ?', "%#{search}%"])
```
Comment (inahsanjuan) [inline app/models/user.rb]: @hironaooo 
This line is enough

File: app/views/users/index.html.erb [hunk 4]
```diff
@@ -1,38 +1,41 @@
...
-                                <h4><%= link_to user.name, user %></h4>
-                                <% unless current_user?(user) %>
-                                    <div id="follow-form">
-                                        <% if current_user.following?(user) %>
-                                            <%= render 'unfollow', user: user %>
-                                        <% else %>
-                                            <%= render 'follow', user: user%>
-                                        <% end %>
-                                    </div>
+<% if @users.nil? %>
```
Comment (inahsanjuan) [inline app/views/users/index.html.erb]: @hironaooo You can use `unless @users.any?`

**Pull request merged**